### ISSUE TITLE
There seems to be an error.

### ISSUE THREAD

**Issue by piano777wzy** [CG8516/PvZA11y#13]
Hello, after I run the tool, there will be a file called Crashlog.txt. There seems to be something wrong with my translation of the contents.
Index was outside the bounds of the array. At PvZA11y.Program.HookProcess ()
At PvZA11y.Program.SafeMain (String [] args)
At PvZA11y.Program.Main (String [] args) System.Diagnostics.Process HookProcess () System.Collections.ListDictionaryInternal
However, it seems to have no effect on playing games, or it may have an impact. I didn't find it.

**Comment by CG8516** [comment]
Thanks for reporting! Looks like there's a potential crash that could happen when the mod starts.  
I just wrote a fix, which will be included in the next update.  

If it was just a one-off thing, or only happens once when you first start the mod, there's nothing to worry about. It won't have any impact on the game/mod.

**Comment by piano777wzy** [comment]
> 感谢您的举报！看起来模组启动时可能会发生崩溃。我刚刚写了一个修复程序，它将包含在下一次更新中。
> 
> 如果这只是一次性的事情，或者在你第一次启动模组时只发生一次，那就没有什么可担心的了。它不会对游戏/模组产生任何影响。

Hello, I just went to observe carefully, first I started the game, then I started mod, at this time Crashlog.txt did not appear, at this time there is no this file, and then I quit the game, when Crashlog.txt appeared.
But it doesn't seem to have any effect on playing games.
It's the same thing in the file.
Index was outside the bounds of the array. At PvZA11y.Program.HookProcess ()
At PvZA11y.Program.SafeMain (String [] args)
At PvZA11y.Program.Main (String [] args) System.Diagnostics.Process HookProcess () System.Collections.ListDictionaryInternal
